src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -132,115 +224,871 @@ static DamageTable D_8092C140 = {
+            } else {
+                this->effectScale = this->effectScale;
+            }
+        } else if (Math_StepToF(&this->steamScale, 0.65f, 0.01625f) == 0) {
+            func_800B9010(&this->actor, NA_SE_EV_ICE_FREEZE - SFX_FLAG);
+        }
+    }
+    EnIk_ArmorDestroyed(this, globalCtx);
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A570.s")
+s32 EnIk_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    EnIk* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A5BC.s")
+    if (this->drawArmorFlags != 0) {
+        if (D_8092C1A8[limbIndex] > 0) {
+            *dList = NULL;
+        }
+    }
+    return false;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A680.s")
+void EnIk_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    EnIk* this = THIS;
+    s32 sp80 = D_8092C1A8[limbIndex];
+    Gfx* tmp;
+    EnIkStruct* ptr;
+    s16 sp76;
+    Vec3f sp68;
+    Vec3f sp5C;
+    Vec3f sp50;
+    Vec3f sp44;
+    MtxF* mtfxPtr;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A754.s")
+    if (this->drawArmorFlags == 0x1) {
+        if (sp80 > 0) {
+            ptr = &this->unk_550[sp80];
+            mtfxPtr = Matrix_GetCurrentState();
+            ptr->unk_04.x = mtfxPtr->mf[3][0];
+            ptr->unk_04.y = mtfxPtr->mf[3][1];
+            ptr->unk_04.z = mtfxPtr->mf[3][2];
+            func_8018219C(mtfxPtr, &ptr->unk_1C, 0);
+            ptr->unk_24 = 1;
+            sp76 = D_8092BFA0[sp80].unk04 + ((Rand_Next() >> 0x13) + this->actor.shape.rot.y);
+            ptr->unk_10.x = Math_SinS(sp76) * 5.0f;
+            ptr->unk_10.y = 6.0f;
+            ptr->unk_10.z = Math_CosS(sp76) * 5.0f;
+            ptr->unk_00 = *dList;
+        }
+        if (limbIndex == IRON_KNUCKLE_LIMB_WAIST) {
+            this->drawArmorFlags |= 0x2;
+        }
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A8D8.s")
+    if (limbIndex == IRON_KNUCKLE_LIMB_ROOT) {
+        Matrix_GetStateTranslation(&this->actor.focus.pos);
+        this->colliderCylinder.dim.pos.x = this->actor.focus.pos.x;
+        this->colliderCylinder.dim.pos.y = this->actor.world.pos.y;
+        this->colliderCylinder.dim.pos.z = this->actor.focus.pos.z;
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ik/func_8092A994.s")
+    if ((limbIndex == IRON_KNUCKLE_LIMB_17) && (this->unk_2F8 != this->counter) &&
+        ((this->actionFunc == EnIk_VerticalAttack) || (this->actionFunc == EnIk_HorizontalDoubleAttack) ||
+         (this->actionFunc == EnIk_SingleHorizontalAttack))) {
+        Math_Vec3f_Copy(&sp68, this->colliderQuad.dim.quad);
+        Math_Vec3f_Copy(&sp5C, &this->colliderQuad.dim.quad[1]);
+        Matrix_MultiplyVector3fByState(&D_8092C1C8, &sp50);
+        Matrix_MultiplyVector3fByState(&D_8092C1D4, &sp44);
```

Comment:
I'm guessing these are in-function static. Can name the quad vertex temps too.

Reply:
I tried in-function static, didn't seem to work (could have done it wrong though)